x-pack/plugins/fleet/kibana.json

```diff
@@ -8,8 +8,8 @@
   "server": true,
   "ui": true,
   "configPath": ["xpack", "fleet"],
-  "requiredPlugins": ["licensing", "data", "encryptedSavedObjects", "navigation", "customIntegrations", "share", "spaces", "security"],
-  "optionalPlugins": ["features", "cloud", "usageCollection", "home", "globalSearch", "telemetry"],
+  "requiredPlugins": ["licensing", "cloud", "data", "encryptedSavedObjects", "navigation", "customIntegrations", "share", "spaces", "security"],
```

Comment:
You should be able to keep `cloud` as optional as long as you also add it to `requiredBundles` below (notice how this is what was done for `home` and `usageCollection`, which are still optional plugins even though the bundles are required for some static imports)

Reply:
I'll adjust this, as well.